apps/api/lib/viewCollection.js

```diff
@@ -11,18 +11,24 @@ export function textFilter({ subject, object, variable }) {
 
   return sparql`
     {
-      ${subject} ${schema.name} ${q}
-    }
-    UNION
-    {
-      ${subject} ${schema.alternateName} ${q}
+      {
+        ${subject} ${schema.name} ${q}
+      }
+      UNION
+      {
+        ${subject} ${schema.alternateName} ${q}
+      }
+      UNION
+      {
+        ${subject} ${schema.author}/${vcard.hasName} ${q}
+      }
+  
+      FILTER( REGEX(str(${q}), "${object.value}", "i") )
     }
     UNION
     {
-      ${subject} ${schema.author}/${vcard.hasUID} ${q}
+      ${subject} ${schema.author}/${vcard.hasUID} "${object.value}"
```

Comment:
In the search for views, I added this separate case to make an exact match of user ID